tasks/deploy_patch_static_service.yml

```diff
@@ -0,0 +1,23 @@
+# Patch services of all services from a single app to the new deployment_stamp
+
+- name: "Patching services with prefix {{ prefix }} for the {{ app.name }} application with deployment_stamp {{ deployment_stamp }}"
+  k8s:
+    force: "{{ force_route | bool }}"
+    definition: "{{ lookup('template', service_template) | from_yaml }}"
+    state: present
+  loop: "{{ static_services }}"
+  loop_control:
+    loop_var: service_template
+  when: static_services | length > 0 and app.settings.is_blue_green_compatible | default(True) == True
+  tags: static_service
+
+- name: "Patching services for the {{ app.name }} application"
+  k8s:
+    force: "{{ force_route | bool }}"
+    definition: "{{ lookup('template', service_template) | from_yaml }}"
+    state: present
+  loop: "{{ static_services }}"
+  loop_control:
+    loop_var: service_template
+  when: static_services | length > 0 and app.settings.is_blue_green_compatible | default(True) == False
+  tags: static_service
```

Comment:
These 2 steps look the same to me and I fail to understand why they can't be written in one. Is it just for logging purposes?

Reply:
Yes, I think it's for logging purpose only ! It's useful to track the progress during a switch.
it was the same thing on deploy_patch_route.yml before this PR.

Reply:
👍